Plot/Plot.gradle

```diff
@@ -17,7 +17,7 @@ dependencies {
     implementation project(':engine-tuple')
     implementation project(':log-factory')
 
-    Classpaths.inheritGroovy(project, 'groovy', 'implementation')
+    Classpaths.inheritGroovy(project, 'groovy', 'api')
```

Comment:
technically true, but I am not sure those are actually usable?

Reply:
My test did not compile when this was listed as `implementation`; I think it has to do w/ the FigureFactory / interface.

Reply:
Huh. Worth tracking down, but ... not today.

Reply:
```
> Task :java-client-session-dagger:compileTestJava FAILED
/home/devin/dev/deephaven/deephaven-core/java-client/session-dagger/src/test/java/io/deephaven/client/ObjectServiceTest.java:101: error: cannot access Closure
        return FigureFactory.figure().plot("Test", t, "X", "Y").show();
                                     ^
  class file for groovy.lang.Closure not found
1 error
```